src/lens/V4Quoter.sol

```diff
@@ -138,4 +143,15 @@ contract V4Quoter is IV4Quoter, BaseV4Quoter {
         uint256 amountIn = params.zeroForOne ? uint128(-swapDelta.amount0()) : uint128(-swapDelta.amount1());
         amountIn.revertQuote();
     }
+
+    function msgSender() external view returns (address) {
```

Comment:
i'm wondering if this should be implemented in MsgSender.sol and it could be an abstract contract? or do you think some people will implement the function in a different way?

Reply:
i think its okay as-is, posm/UR implement `msgSender()` through _ReentrancyLock_ while quoter implements it directly via _Locker_

so theres a possibility someone implements it different, i.e. with contract storage for some reason